pkg/actions/tools/ffmpeg/codec.go

```diff
@@ -11,18 +11,48 @@ import (
 //
 //	4gv (4GV (Fourth Generation Vocoder))
 //	4xm (4X Movie)
-func ActionCodecs() carapace.Action {
-	return carapace.ActionExecCommand("ffmpeg", "-hide_banner", "-codecs")(func(output []byte) carapace.Action {
-		lines := strings.Split(string(output), "\n")
-		r := regexp.MustCompile(`^.{7} (?P<codec>[^ ]+) +(?P<description>.*)$`)
-
-		vals := make([]string, 0)
-		for _, line := range lines[10 : len(lines)-1] {
-			if r.MatchString(line) {
-				matches := r.FindStringSubmatch(line)
-				vals = append(vals, matches[1], matches[2])
+func ActionCodecs(codecType rune) carapace.Action {
+	return carapace.ActionExecCommand("ffmpeg", "-hide_banner", "-codecs")(func(outputCodecs []byte) carapace.Action {
+		encodersCmd := carapace.ActionExecCommand("ffmpeg", "-hide_banner", "-encoders")
+		return encodersCmd(func(outputEncoders []byte) carapace.Action {
+			linesCodecs := strings.Split(strings.TrimSpace(string(outputCodecs)), "\n")
+			linesEncoders := strings.Split(strings.TrimSpace(string(outputEncoders)), "\n")
+
+			r := regexp.MustCompile(`^ (?P<prefix>.{6}) (?P<codec>\w+) +(?P<description>.*)$`)
+
+			vals := make(map[string]string)
+
+			// copy
+			vals["copy"] = "copy the codec of the input"
+
+			// Parse codecs
+			for _, line := range linesCodecs {
+				if r.MatchString(line) {
+					matches := r.FindStringSubmatch(line)
+					prefix := matches[1]
+					if (codecType == '\u0000' || byte(codecType) == prefix[2]) && 'E' == prefix[1] {
```

Comment:
All the `\u0000` should use a constant. What it does now in unclear

Reply:
It is just whining about the order of the check ([ST1017](https://staticcheck.dev/docs/checks/#ST1017)):

```go
-					if (codecType == '\u0000' || byte(codecType) == prefix[2]) && 'E' == prefix[1] {
+					if (codecType == '\u0000' || byte(codecType) == prefix[2]) && prefix[1] == 'E' {
```

Reply:
Reordered the conditions.

Reply:
> All the `\u0000` should use a constant. What it does now in unclear

I don't think the point I raised was understood.

I was not talking about a CI complaining about this line.

I was asking to consider adding a constant for \u0000and give to that constant a meaningful name, and a decent comment

Reply:
Ah yes, that is still a bit unclear what it represents.

Reply:
I'll get this in later, just leave it as it is.